src/mame/heathkit/h19.cpp

```diff
@@ -32,16 +34,15 @@ class h19_state : public driver_device
 {
 public:
 	h19_state(const machine_config &mconfig, device_type type, const char *tag)
-			: driver_device(mconfig, type, tag)
-			, m_tlb(*this, "tlb")
+		: driver_device(mconfig, type, tag),
+		 	m_tlb(*this, "tlb")
```

Comment:
Please either use delimiter-at-end or delimiter-at-beginning for intialiser lists.

Either this:
```c++
	h19_state(const machine_config &mconfig, device_type type, const char *tag) :
		driver_device(mconfig, type, tag),
		m_tlb(*this, "tlb")
```

Or this:
```c++
	h19_state(const machine_config &mconfig, device_type type, const char *tag)
		: driver_device(mconfig, type, tag)
		, m_tlb(*this, "tlb")
```